src/frontend/src/content/docs/ja/whats-new/aspire-13-1.mdx

```diff
@@ -0,0 +1,591 @@
+import genaiVisualizerTooldefinitions from '@assets/whats-new/genai-visualizer-tooldefinitions.png';
+
+Aspire 13.1 は、Aspire 13 で確立されたポリグロット基盤をさらに発展させ、CLI 体験の大幅な改善、AI コーディングエージェント向けの MCP（Model Context Protocol）サポートの強化、ダッシュボードの改良、Azure へのデプロイの簡素化を提供します。本リリースは、開発者の生産性向上と、最新の AI 支援開発ワークフローとのシームレスな統合に重点を置いています。
+
+このリリースで導入される内容は次のとおりです:
+
+- **AI コーディングエージェント向け MCP**: 新しい `aspire mcp init` コマンドと、AI アシスタントが統合を発見し Aspire アプリケーションと連携するための MCP ツール。
+- **CLI の強化**: チャネルの永続化、自動インスタンス検出、インストールパスのオプション。
+- **ダッシュボードの改善**: 専用の Parameters タブと生成 AI ビジュアライザーの強化。
+- **Azure マネージド Redis**: `AddAzureRedisEnterprise` から `AddAzureManagedRedis` へ名称変更し、検証を改善。
+- **コンテナー レジストリ サポート**: 汎用コンテナー レジストリ向けの新しい `ContainerRegistryResource`。
+- **JavaScript の改善**: 新しいフロントエンド スターター テンプレート、Vite の HTTPS 設定、pnpm の修正。
+- **DevTunnels の安定化**: [📦 Aspire.Hosting.DevTunnels](https://www.nuget.org/packages/Aspire.Hosting.DevTunnels) が安定版パッケージになりました。
+- **TLS 終端サポート**: コンテナーやリソースに HTTPS 証明書を設定するための新しい API。
+- **Azure リソースの接続プロパティ**: Azure リソースの接続文字列に対するポリグロット対応。
+
+**要件:**
+
+- [.NET 10 SDK 以降](https://dotnet.microsoft.com/download/dotnet/10.0)
+
+フィードバックやご質問、Aspire への貢献にご興味がある場合は [<Icon name="github" class='d-inline' /> GitHub](https://github.com/dotnet/aspire) でコラボレーションするか [<Icon name="discord" class='d-inline' /> Discord](https://aka.ms/aspire-discord) に参加して、チームやコミュニティメンバーと交流してください。
+
+## 🆙 Aspire 13.1 へのアップグレード
+
+<Aside type="caution">
+Aspire 13.1 には Azure Redis API に関する破壊的変更が含まれています。アップグレード前に必ず [破壊的変更](#%EF%B8%8F-破壊的変更) セクションを確認してください。
+</Aside>
+
+Aspire 13.1 へアップグレードする最も簡単な方法は、`aspire update` コマンドを使用することです:
+
+1. [`aspire update` コマンド](/ja/reference/cli/commands/aspire-update/) を使用して Aspire プロジェクトを更新します:
+
+    ```bash title="Aspire CLI — すべての Aspire パッケージを更新"
+    aspire update
+    ```
+
+2. Aspire CLI 自体を更新します:
+
+    ```bash title="Aspire CLI — CLI を更新"
+    aspire update --self
+    ```
+
+<Aside type="note">
+`aspire update --self` を実行してチャネルを選択すると、その選択はグローバルに保存されます。以後の `aspire new` や `aspire init` コマンドでは、このチャネルが既定で使用されます。
+</Aside>
+
+## 🤖 AI コーディングエージェント向け MCP
+
+Aspire 13.1 では、MCP（Model Context Protocol）との統合により、AI コーディングエージェントを包括的にサポートします。お好みの AI コーディングツールを設定することで、Aspire アプリケーションを理解し、操作できるようになります。
+
+### プロジェクト用に MCP を初期化する
+
+新しい `aspire mcp init` コマンドは、開発環境を検出し、対応する AI コーディングエージェント向けに MCP サーバーを設定します:
+
+```bash title="Bash — MCP 構成を初期化"
+aspire mcp init
+```
+
+<LearnMore>
+コマンド リファレンス: [`aspire mcp init`](/ja/reference/cli/commands/aspire-mcp-init/).
+</LearnMore>
+
+このコマンドでは、簡潔な 2 ステップの選択プロセスが表示されます:
+
+```bash
+# どのエージェント環境を設定しますか？
+Which agent environments do you want to configure?
+[ ] Configure VS Code to use the Aspire MCP server
+[ ] Configure GitHub Copilot CLI to use Aspire MCP server
+[ ] Configure Claude Code to use Aspire MCP server
+[ ] Configure Open Code to use Aspire MCP server
+
+# 追加で有効にするオプションはどれですか？
+Which additional options do you want to enable?
+[ ] Create an agent instructions file (AGENTS.md)
+[ ] Configure Playwright MCP server
```

Comment:
To my knowledge, the Aspire CLI supports translations here too, have you considered pulling those strings? For example, after a quick look at the Aspire source, I found this: https://github.com/dotnet/aspire/blob/v13.1.0/src/Aspire.Cli/Resources/xlf/McpCommandStrings.ja.xlf - there are many more as well.